frontend/src/pages/Admin/Subcontractors/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Noted, technically I went out of scope because my new changes were causing a bug in the subcontractors which in turn wouldn't render the app in the dom :(

Reply:
just fyi, this is not a thing it doesn't have to be named this, however a good idea, it is more important in your commit to have both pr ie DESCW-275 DESCW-279.  
Having said this we should strive to keep pr linked to one jira ticket, but sometimes as we all know is not always possible.

Reply:
I realize that it's not mandatory to name your branches accordingly but it has been a matter of convention. I think the salient point is just that if we're committing work tied to multiple tickets that we provide transparency by noting the tickets somewhere(ie. CHANGELOG).

That said, Adam and I chatted and he was just updating other code that broke as a result of implementing DESCW-275.